package.json

```diff
@@ -4,15 +4,15 @@
   "displayName": "Mezzurite",
   "author": "Microsoft Corporation",
   "license": "MIT",
-  "version": "0.0.0",
+  "version": "2.1.0",
   "repository": {
     "url": "https://github.com/Microsoft/vscode-mezzurite",
     "type": "git"
   },
   "bugs": {
     "url": "https://github.com/Microsoft/vscode-mezzurite"
   },
-  "publisher": "vscode-samples",
+  "publisher": "mezzurite-devs",
```

Comment:
Is there a reason to bundle these changes together with the package-lock.json changes?

Reply:
It's fine, but it seemed more appropriate as a separate PR.

Reply:
More for convenience, but we can separate them as distinct PRs if needed. As discussed offline, the future of this extension is still up in the air, so the intent was trying to keep activity on this repository low until that decision is made.